.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Each line is a file pattern followed by one or more owners.
```

Comment:
maybe specify that the order matters, and go from most global to most restrictive (ie later patterns take precedence over earlier ones) ?